exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/handlers/ViewHandler.java

```diff
@@ -49,6 +50,24 @@ public ViewHandler(SqlHandlerConfig config) {
     this.context = config.getContext();
   }
 
+  /**
+   * If view to be dropped is in default temporary workspace, checks if it's a temporary table or not.
+   *
+   * @param schema view schema
+   * @param viewName view name to be created
+   * @return true is object to be created is temporary table, false otherwise
+   */
+  protected boolean isTemporaryTable(AbstractSchema schema, String viewName) {
```

Comment:
Not sure why we are checking if a view is a temporary table? Or, should viewName be tableName?